extension/src/time_vector.rs

```diff
@@ -22,6 +22,9 @@ use crate::raw::bytea;
 
 pub use toolkit_experimental::{Timevector, TimevectorData};
 
+// Bit flags stored in Timevector flags
+const FLAG_HAS_NULLS : u8 = 0x01;
```

Comment:
The [bitflags](https://docs.rs/bitflags/latest/bitflags/) library may be useful for this